src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
```

Comment:
card should have `padding: 32px 16px 16px;` for all sides to avoid margin-right/margin-left for inner elements